plonky_block_proof_gen/src/prover_state.rs

```diff
@@ -72,10 +73,11 @@ impl ProverStateBuilder {
     // TODO: Consider adding async version?
     /// Instantiate the prover state from the builder. Note that this is a very
     /// expensive call!
-    pub fn build(self) -> ProverState {
-        info!("Initializing Plonky2 aggregation prover state (This may take a while)...");
+    pub fn build(self, verbose: bool) -> ProverState {
+        if verbose {
+            info!("Initializing Plonky2 aggregation prover state (This may take a while)...");
```

Comment:
I'm not a fan of this, but I wanted to silence the `info!(...)` so as to reuse the same code for the verifier side, to reduce code duplication. Not sure that's the neatest approach though..

Reply:
Yeah... I'm personally ok with this, although I am not a huge fan of passing a bool as an arg for verbosity here. As an alternative, you could downgrade these to `debug!`s as well. I might be wrong, but I think `info` is often the default level used for trace levels.

Reply:
I'll just revert it back. It's technically not wrong to get both logs, as the `VerifierCircuitBuilder` does generate the prover circuits first before extracting the verifier circuit data anyway, and I hate this boolean floating around

Reply:
`info` is the default and I think it's best to stick to the rust-standard way of tweaking verbosity, i.e. using `RUST_LOG=module=LEVEL`.